tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1492,6 +1492,29 @@ pub struct Identity {
 
 // === impl Subscribe ===
 
+#[derive(Clone, Copy)]
+pub(crate) struct NoneLayerMarker(());
+pub(crate) static NONE_LAYER_MARKER: NoneLayerMarker = NoneLayerMarker(());
```

Comment:
also, i think the static doesn't _need_ to be `pub(crate)`, since it's only referenced inside this file.
```suggestion
static NONE_LAYER_MARKER: NoneLayerMarker = NoneLayerMarker(());
```